libtest/testutils.c

```diff
@@ -212,7 +212,7 @@ assert_gdouble_non_fatal(gdouble actual, gdouble expected, const gchar *error_me
 {
   va_list args;
 
-  if (actual == expected)
+  if (actual <= expected)
```

Comment:
You cannot simple replace float equal with less equal or larger equal.

Reply:
We can do following comparison,
```
if (fabs(actual - expected) < 1e-12)
```